### PR TITLE
New content wrapper for webpages

### PR DESCRIPTION
**Pull request by Itrulia**
Added new content wrapper for everything that should be narrow, else use wide container

Also changed:

1. Styling of buttons
2. Quote block design
3. Bajour primary color
4. Gutter styles of paragraphs
5. line heights
6. fonts of body text
7. Bajour hide last 6col in blocks
8. Bajour connect archive with data
9. Bajour fix social media logos
10. Moved author links to website builder & added mapping of link to icon

### CODE REVIEW

File: .github/workflows/on-demand-deploy-vercel.yml
Comment (Itrulia) [inline .github/workflows/on-demand-deploy-vercel.yml:1]: Hm how did these changes come in, i j ust merged from master

File: libs/block-content/website/src/lib/image/image-block.tsx [hunk 1]
```diff
@@ -12,18 +13,27 @@ export const isImageBlock = (block: Block): block is ImageBlockType =>
...
+  gap: ${({theme}) => theme.spacing(2)};
   margin: 0;
   max-width: 100%;
 `
 
+export const ImageBlockCaption = styled('figcaption')``
+
+const imageStyles = css`
+  object-fit: cover;
+  aspect-ratio: 1.85611511;
```
Comment (antkiewiczk) [inline libs/block-content/website/src/lib/image/image-block.tsx:26]: do we want this many decimal places? wouldn't maximum of two or three be enough?

**Review by antkiewiczk — COMMENTED** [review]
Please try and split it up next time, there's 180 files to review. This should be at least 3 separate PRs. Otherwise looking good, there are still some conflicts to resolve

File: libs/block-content/website/src/lib/image/image-block.tsx (hunk 1 above)
Comment (Itrulia) [reply libs/block-content/website/src/lib/image/image-block.tsx:26]: Just c&p from rons comments in the design

**Pull request merged**